File: aterm-csharp/ATerm/Interfaces/ATermInt.cs
using System;

namespace aterm
{
	/// <summary>
	/// This interface describes the functionality of an ATermInt
	/// </summary>
	public interface ATermInt : ATerm
	{
		/**
	     * Gets the integer value from this term.
		 *
		 * @return the integer value from this term.
		 *
		 */
		int getInt();		
	}
}

File: aterm/java/aterm/ATermInt.java
package aterm;

/**
 * This interface describes the functionality of an ATermInt
 * 
 * @author Hayco de Jong (jong@cwi.nl)
 * @author Pieter Olivier (olivierp@cwi.nl)
 * @version 0.1, Fri Jan 28 10:19:58 MET 2000
 */
public interface ATermInt extends ATerm {

    /**
     * Gets the integer value from this term.
     *
     * @return the integer value from this term.
     *
     */
    public int getInt();
}
